Model/statemachine.py

```diff
@@ -248,7 +248,7 @@ def __init__(
         self.I_version = i_version
         self.cancel_coin_selection = cancel_coin_selec
 
-        self.vb_coins_count = 8
+        self.vb_coins_count = 6
```

Comment:
I remember we discussed it off band but it would be good to keep a rationale for such changes..